scene/2d/tile_map.cpp

```diff
@@ -30,6 +30,8 @@
 
 #include "tile_map.h"
 
+#include "area_2d.h"
```

Comment:
This is not covered by [our guidelines](http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes) so far, but I wonder if we should enforce that header includes should always use the "absolute" path from project root, e.g. `"scene/2d/area_2d.h"` to avoid confusion and have a better alphabetical ordering by path.

This would not apply to the .cpp file's own .h though.

Reply:
Fixed!